TRON Core Devs Meetings/Meeting 48.md

```diff
@@ -0,0 +1,184 @@
+  - Discussion on Nonce Implementation for EIP-7702 Compatibility ([TIP-7702](https://github.com/tronprotocol/tips/issues/728))
+  - Syncing the client release workflow guidelines
+
+
+### Detail
+
+* **Murphy**
+
+    Welcome, everyone, to the 48th Core Devs Community Meeting. We have four topics on the agenda today. First, I'd like to invite Neo to give us an update on the development progress of version 4.8.1.
+
+**Syncing the development progress of v4.8.1**
+
+* **Neo**
+
+    Version 4.8.1 is currently in the testing phase, and we haven't found many issues so far. The focus of our testing is on compatibility with the ARM platform, which might require the developers to conduct several rounds of regression testing. We have an initial draft of the Release Note, and we plan to review the content again within the next two weeks before publishing it to the PM repository.
+
+* **Patrick**
+
+    So, is there any change to the release timeline? Like a specific date?
+
+* **Neo**
+
+    The release is tentatively planned for November, but the specific date isn't set yet, as it depends on the testing progress.
+    
+* **Murphy**
+
+    Alright. If there are no other questions, let's move on to the second topic. I'd like to ask Aiden to introduce the latest progress on the changes to the `SELFDESTRUCT` instruction.
+
+
+**Progress Sync on `SELFDESTRUCT` Opcode Modification**
+
+- **Aiden**
+    
+    I'll provide an update on the changes to the SELFDESTRUCT opcode. The core logic of the change is: when the contract creation and the call to the `SELFDESTRUCT` opcode are not in the same transaction, this opcode will no longer delete the contract state but will only transfer assets. Additionally, the energy cost for the `SELFDESTRUCT` opcode has been adjusted from 0 to 5000.
+    
+    On the TRON network, this opcode processes all native assets, including TRC-10 tokens, voting rewards, and staked resources. It first automatically withdraws all claimable rewards and cancels existing votes, then transfers these assets along with the TRX and TRC-10 balances to the target address. For resources staked for the account's own use, both the resources themselves and their associated usage data are also transferred to the target.
+
+    This change has two main compatibility impacts. First, the `create2` opcode can no longer recreate a contract at a self-destructed address because the account data is no longer fully deleted. Second, burning TRX by self-destructing to the same address is no longer possible, as the balance will simply remain unchanged.
+
+    Lastly, we've compiled statistics on Mainnet contracts that use the `SELFDESTRUCT` opcode. Among them, only 3 hold assets greater than 100,000 TRX, and 12 hold more than 10,000 TRX. Overall, the direct impact appears to be minimal. I will further analyze the relevant transaction data and provide a more detailed impact analysis in our next meeting.
```

Comment:
Per commets https://github.com/tronprotocol/tips/issues/765#issuecomment-3101228049, it is 4 address,